src/psyclone/dynamo0p3.py

```diff
@@ -4045,9 +4054,12 @@ def __init__(self, fine_arg, coarse_arg):
         # Generate name for ncell variables
         self.ncell_fine = symtab.symbol_from_tag(
             "ncell_{0}".format(fine_arg.name)).name
-        # No. of fine cells per coarse cell
-        self.ncellpercell = symtab.symbol_from_tag(
-            "ncpc_{0}_{1}".format(fine_arg.name, coarse_arg.name)).name
+        # No. of fine cells per coarse cell in x
+        self.ncellpercellx = symtab.symbol_from_tag(
+            "ncpc_{0}_{1}_x".format(fine_arg.name, coarse_arg.name)).name
+        # No. of fine cells per coarse cell in y
+        self.ncellpercelly = symtab.symbol_from_tag(
+            "ncpc_{0}_{1}_y".format(fine_arg.name, coarse_arg.name)).name
```

Comment:
This is not the fault of this PR, but `pylint` reports
```
dynamo0p3.py:7167:0: C0330: Wrong hanging indentation (remove 7 spaces).
                                          prev_node_name)))
                                   |      ^ (bad-continuation)
```
Would you mind fixing this?

Reply:
I couldn't get my pylint to complain about that (2.7.4) but I think (hope) I've fixed it.

Reply:
It's fine now, thanks.